core/templates/dev/head/components/answer_group_editor_directive.html

```diff
@@ -93,9 +93,9 @@
                   suppress-warnings="suppressWarnings()">
   </outcome-editor>
 
-  <br ng-if="answerGroupIsDefault">
+  <br ng-if="answerGroupIsDefault && isCorrectnessFeedbackEnabled">
```

Comment:
I wonder if you can bring this br inside the div below, or better, style the div below so that it has a sufficient margin-top?

Otherwise, we won't remember to keep parity between this tag and the next, which isn't great for maintainability.

Reply:
Done!